components/tissue-score-bar.js

```diff
@@ -18,8 +18,7 @@ ChartJS.register(
   BarElement,
   Title,
   Tooltip,
-  Legend,
-  Datalabels
+  Legend
```

Comment:
I haven’t worked with chartjs in a long time, and never with react-chartjs-2, so I really don’t know for sure. But it looks like `ChartJS.register()` is intended just for global configurations. They describe an instance-based configration https://chartjs-plugin-datalabels.netlify.app/guide/getting-started.html#registration. I don’t know if that works for you, but I feel like the last `ChartJS.register` that happens to get called during a reload would get used globally. The last `ChartJS.register` called won’t necessarily be the one above the component it’s supposed to apply to.